File: Final-Bench/benchmark.js
console.log("Starting intensive benchmark...");

for (let run = 1; run <= 5; run++) {
  const start = process.hrtime.bigint();

  let sum = 0n;
  for (let i = 0n; i < 100000000n; i++) {
    sum += i;
  }

  const elapsed = Number(process.hrtime.bigint() - start) / 1e9;

  console.log(`Run: ${run} Sum: ${sum} Time: ${elapsed.toFixed(3)} s`);
}

console.log("Benchmark complete!");

File: Final-Bench/benchmark.py
import time

print("Starting intensive benchmark...")

for run in range(1, 6):
    start = time.perf_counter()

    total_sum = 0
    for i in range(100_000_000):
        total_sum += i

    elapsed = time.perf_counter() - start
    print(f"Run: {run} Sum: {total_sum} Time: {elapsed:.3f} s")

print("Benchmark complete!")
